app/lib/scorecard/backend.dart

```diff
@@ -124,9 +124,11 @@ class ScoreCardBackend {
         return null;
       }
     }
-    final cacheEntry = onlyCurrent || showSandboxedOutput
+    final cacheEntry = onlyCurrent
         ? null
-        : cache.scoreCardData(packageName, packageVersion);
+        : (showSandboxedOutput
+            ? cache.scoreCardData2(packageName, packageVersion)
```

Comment:
I think we need to also store something in `scoreCardData2`.

Also we probably need purge from within the task backend.

Reply:
The store is automatic in this method.
I've added calls for the purge.

Reply:
> The store is automatic in this method.

The `    if (showSandboxedOutput) {` will return directly, without storing something in the cache.

Reply:
fixed, PTAL